src/libpriv/rpmostree-origin.c

```diff
@@ -239,10 +239,10 @@ rpmostree_origin_classify_refspec (RpmOstreeOrigin      *origin,
     *out_refspecdata = origin->cached_refspec;
 }
 
-const char *
+char *
 rpmostree_origin_get_rojig_version (RpmOstreeOrigin *origin)
```

Comment:
There's a caller in the upgrader which uses this that I think needs to be switched over to `g_autofree`.